src/main/scala/io/viash/functionality/resources/NextflowScript.scala

```diff
@@ -48,26 +54,36 @@ case class NextflowScript(
   def generateInjectionMods(argsMetaAndDeps: Map[String, List[Argument[_]]], config: Config): ScriptInjectionMods = {
     val configPath = s"$$targetDir/${config.functionality.namespace.getOrElse("namespace")}/${config.functionality.name}/.config.vsh.yaml"
 
-    val dependenciesAndDirNames = config.functionality.dependencies.map(d => (d, d.repository.toOption.get.name))
+    val (localDependencies, remoteDependencies) = config.functionality.dependencies
+      .partition(d => d.isLocalDependency)
+    val localDependenciesStrings = localDependencies.map{ d =>
+      // Go up the same amount of times that the namespace strategy specifies during the build step so we can go down to the local dependency folder
+      val up = ViashNamespace.targetOutputPath("", "..", config.functionality.namespace.map(ns => ".."), "..")
```

Comment:
`config.functionality.namespace.map(ns => "..")` is wrong when a namespace is nested

**If** you have the path of the local dependency and the path of the current config dir available, you could use relativize:

```scala
val p1 = Paths.get("/home/rcannood/Downloads")
val p2 = Paths.get("/home/rcannood/Documents/foo.xlsx")

p1.relativize(p2)
```

    val res1: java.nio.file.Path = ../Documents/foo.xlsx